RF24.cpp

```diff
@@ -1606,7 +1614,6 @@ void RF24::openWritingPipe(const uint8_t* address)
     // expects it LSB first too, so we're good.
     write_register(RX_ADDR_P0, address, addr_width);
     write_register(TX_ADDR, address, addr_width);
-    memcpy(pipe0_writing_address, address, addr_width);
```

Comment:
This line needs to be added back in I believe.

memcpy(pipe0_writing_address, address, addr_width);

Reply:
Done. Not sure why I removed it. 🤷🏼‍♂️